builddefs/generic_features.mk

```
@@ -36,6 +36,7 @@ GENERIC_FEATURES = \
     HAPTIC \
     KEY_LOCK \
     KEY_OVERRIDE \
+	KEY_INTERRUPT \
```

something something make is stupid and picky, so lets not mix stuff
```suggestion
    KEY_INTERRUPT \
```